screenpy/__version__.py

```diff
@@ -1,16 +1,13 @@
-"                 ____                           ____"
-#                / ___|  ___ _ __ ___  ___ _ __ |  _ \ _   _
-#                \___ \ / __| '__/ _ \/ _ \ '_ \| |_) | | | |
-#                 ___) | (__| | |  __/  __/ | | |  __/| |_| |
-#                |____/ \___|_|  \___|\___|_| |_|_|    \__, |
-#                                                      |___/
+r"""
+ ____                           ____
+/ ___|  ___ _ __ ___  ___ _ __ |  _ \ _   _
+\___ \ / __| '__/ _ \/ _ \ '_ \| |_) | | | |
+ ___) | (__| | |  __/  __/ | | |  __/| |_| |
+|____/ \___|_|  \___|\___|_| |_|_|    \__, |
+                                      |___/
+"""
```

Comment:
I had wanted this to look centered on the page.

Reply:
Ohhhh!  oops. I'll fix it.